.travis.yml

```diff
@@ -16,9 +16,13 @@ env:
     - TEST_BLACKLIST=moveit,moveit_ros,moveit_runtime,moveit_ros_perception  # mesh_filter_test fails due to broken Mesa OpenGL
   matrix:
     - ROS_DISTRO=kinetic ROS_REPO=ros              TEST=clang-format
-    - ROS_DISTRO=melodic ROS_REPO=ros              UPSTREAM_WORKSPACE=moveit.rosinstall
     - ROS_DISTRO=melodic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=moveit.rosinstall
     - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=moveit.rosinstall
```

Comment:
can we unify these two ``matrix:`` sections into one? if not, please add comment explaining why they are different.

seems like we could move all the tests to the below section with ``compiler: gcc``

Reply:
This cannot be merged. As we want to use a different compiler (clang), we need to use the include/matrix section, which allows to add individual entries to the matrix.